dissect/target/volume.py

```diff
@@ -22,6 +22,8 @@
 """A lazy import of :mod:`dissect.target.volumes.vmfs`."""
 md = import_lazy("dissect.target.volumes.md")
 """A lazy import of :mod:`dissect.target.volumes.md`."""
+ddf = import_lazy("dissect.target.volumes.ddf")
```

Comment:
I tried to open a RAID-0 target consisting of 2 DDF RAID-0 disks with EXT4 trough the multirawloader but it does not seem to work? Is there some wiring missing somewhere?

`target-query s1.img+s2.img -f users`

```
s1.img+s2.img: Attempting to use loader: <lazyattr dissect.target.loaders.multiraw.MultiRawLoader loaded=True> [dissect.target.target] 
2023-10-18T14:42:59.404293Z [debug    ] <Target s1.img+s2.img>: LVM volumes found: [<Volume name=None size=10737418240 fs=None>, <Volume name=None size=10737418240 fs=None>] [dissect.target.target] 
2023-10-18T14:42:59.404357Z [debug    ] <Target s1.img+s2.img>: Encrypted volumes found: [] [dissect.target.target] 
2023-10-18T14:42:59.411051Z [debug    ] <Target s1.img+s2.img>: Opened LVM: <DdfVolumeSystem serial=None> [dissect.target.target] 
2023-10-18T14:42:59.471100Z [warning  ] <Target s1.img+s2.img>: Can't identify filesystem: <Volume name='0' size=21407727616 fs=None> [dissect.target.target]
```

Reply:
Can you share these files with me for debugging?